docs/src/architecture/state.md

```diff
@@ -61,14 +61,14 @@ ToDo: Describe and specify life-time restrictions for notes on the database.
 
 ### Nullifier database
 
-Nullifiers provide information on whether a specific note has been consumed yet. Nullifiers are stored in a Sparse Merkle Tree, which maps nullifiers to block heights at which they were created.  Using this, one can check and prove that a given nullifier is not in the database. There will be one tree per epoch (~3 months), and Miden nodes store always at least the last two trees. However, the roots of the old trees are still stored.
+Nullifiers provide information on whether a specific note has been consumed yet. Nullifiers are stored in a Sparse Merkle Tree, which maps nullifiers to `0` or `1`. Using this, one can check and prove that a given nullifier is not in the database. 
```

Comment:
Should we link here to the Tiered Sparse Merkle tree section?

Reply:
yes, we should. Thanks